google/cloud/apigee/registry/applications/v1alpha1/registry_conformance_report.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+
+package google.cloud.apigee.registry.applications.v1alpha1;
+
+import "google/api/field_behavior.proto";
+import "google/cloud/apigee/registry/applications/v1alpha1/registry_styleguide.proto";
+import "google/cloud/apigee/registry/applications/v1alpha1/registry_lint.proto";
+
+option java_package = "com.google.cloud.apigee.registry.applications.v1alpha1";
+option java_multiple_files = true;
+option java_outer_classname = "RegistryConformanceReportProto";
+option go_package = "github.com/apigee/registry/rpc;rpc";
+
+// ConformanceReport describes how well an API Spec or a series of
+// API Specs conform to a specific API Style guide.
+message ConformanceReport {
+    // Resource name of the conformance report.
+    string name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // Name of the style guide that this report pertains to.
+    string styleguide_name = 2 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // A list of guideline report groups.
+    repeated GuidelineReportGroup guideline_report_groups = 3;
+}
+
+// GuidelineReport describes how well an API Spec or a series of
+// API Specs conform to a guideline within an API Style Guide.
+message GuidelineReport {
+    // Name of the guideline that this report pertains to.
+    string guideline_name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // A list of rule report groups.
+    repeated RuleReportGroup rule_report_groups = 2;
+}
+
+// RuleReport provides information and feedback on a rule that 
+// a spec breaches within a guideline on an API Style Guide.
+message RuleReport {
+    // Name of the rule that the spec breaches.
+    string rule_name = 1 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // Resource name of the spec that the rule was breached on.
+    string spec_name = 2 [
+        (google.api.field_behavior) = REQUIRED
+    ];
+
+    // URI of the Rule object on the style guide.
```

Comment:
As we discussed in-person, let's remove this field.

Reply:
Done!